dataland-frontend/src/frameworks/gdv/ViewConfig.ts

```diff
@@ -0,0 +1,1804 @@
+            valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes => {
+              const mappings = {
+                Nein: "Nein",
+                JaAufsichtsrat: "Ja, Aufsichtsrat",
+                JaGeschaeftsleitung: "Ja, Geschäftsleitung",
+                JaAufsichtsratUndGeschaeftsleitung: "Ja, Aufsichtsrat und Geschäftsleitung",
+              };
+              return formatStringForDatatable(
+                dataset.allgemein?.fuehrungsstandards?.anreizmechanismenFuerDasManagementSoziales
+                  ? getOriginalNameFromTechnicalName(
+                      dataset.allgemein?.fuehrungsstandards?.anreizmechanismenFuerDasManagementSoziales,
+                      mappings,
+                    )
+                  : "",
+              );
+            },
+          },
+        ],
+      },
+      {
+        type: "section",
+        label: "Rechtsstreitigkeiten",
+        expandOnPageLoad: false,
+        shouldDisplay: (): boolean => true,
+        children: [
+          {
+            type: "cell",
+            label: "ESG-bezogene Rechtsstreitigkeiten",
+            explanation:
+              "Ist das Unternehmen in laufende bzw. war das Unternehmen in den letzten 3 Jahren in abgeschlossenen Rechtsstreitigkeiten im Zusammenhang mit ESG involviert?",
+            shouldDisplay: (dataset: GdvData): boolean => dataset.general?.masterData?.berichtsPflicht == "Yes",
+            valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes =>
+              formatYesNoValueForDatatable(dataset.allgemein?.rechtsstreitigkeiten?.esgBezogeneRechtsstreitigkeiten),
+          },
+          {
+            type: "cell",
+            label: "Rechtsstreitigkeiten mit Bezug zu E",
+            explanation: 'Haben bzw. hatten die Rechtsstreitigkeiten Bezug zu "E"',
+            shouldDisplay: (dataset: GdvData): boolean =>
+              dataset.allgemein?.rechtsstreitigkeiten?.esgBezogeneRechtsstreitigkeiten == "Yes",
+            valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes =>
+              formatYesNoValueForDatatable(dataset.allgemein?.rechtsstreitigkeiten?.rechtsstreitigkeitenMitBezugZuE),
+          },
+          {
+            type: "cell",
+            label: "Status zu E",
+            explanation: 'Sind die Rechtsstreitigkeiten mit Bezug zu "E" noch offen oder bereits geklärt?',
+            shouldDisplay: (dataset: GdvData): boolean =>
+              dataset.allgemein?.rechtsstreitigkeiten?.rechtsstreitigkeitenMitBezugZuE == "Yes",
+            valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes => {
+              const mappings = {
+                Offen: "offen",
+                Geklaert: "geklärt",
+              };
+              return formatStringForDatatable(
+                dataset.allgemein?.rechtsstreitigkeiten?.statusZuE
+                  ? getOriginalNameFromTechnicalName(dataset.allgemein?.rechtsstreitigkeiten?.statusZuE, mappings)
```

Comment:
What exactly is mapped here? mappings has offen and geklärt, but those are also the options of the StatusZuE Enums? Is this to display Umlaute?

Reply:
You don't want "Geklaert" to appear on the view page. And you don't want "geklärt" to be written into the DB.
That's why frontend-values and technical values are different, and they are connected via these mappings.

Reply:
ok